index.html

```diff
@@ -107,9 +106,14 @@ <h3 class="text-center text-md-left white pb-3">
 							A differential privacy toolkit for analytics and machine learning
 						</h3>
 						<p class="text-center text-md-left pb-4 alpha light">
-							WhiteNoise uses state-of-the-art differential privacy (DP)
-							techniques to inject noise to data, to prevent disclosure of
-							sensitive information and manage exposure risk.
+							This toolkit uses state-of-the-art differential privacy (DP) techniques 
+							to inject noise into data, to prevent disclosure of sensitive information 
+							and manage exposure risk. 
+
+<br>
+<br>
+
+Please note that a previous version of this website referred to the name of the toolkit as WhiteNoise of which we are in the process of <a href="https://projects.iq.harvard.edu/opendp/blog/building-inclusive-community">renaming</a>
```

Comment:
I believe the language we've been given to use is:
[**Please note that we are renaming the toolkit and will be introducing the new name in the coming weeks.**](https://projects.iq.harvard.edu/opendp/blog/building-inclusive-community)